### ISSUE TITLE
`removeItem` & `getItem` should not throw Error when getting data by key not found

### ISSUE THREAD

**Issue by retyui** [robwalkerco/redux-persist-filesystem-storage#47]
```js
// expected 100% compatibility with AsyncStorage

removeItem('not-exist') // should not throw error
getItem('not-exist') // should return null
```
![image](https://user-images.githubusercontent.com/4661784/82154536-44bd2c80-9877-11ea-9439-78b2353495b7.png)

![image](https://user-images.githubusercontent.com/4661784/82154590-864dd780-9877-11ea-877c-c639dd84dc3a.png)

```js
// where LO:
global.LO = require('redux-persist-filesystem-storage').default;
```

**Comment by robwalkerco** [comment]
Thanks for reporting the issue.
Please open a PR with a suggested solution.

**Comment by retyui** [comment]
ignore

**Comment by retyui** [comment]
@robwalkerco Thank for good issue solving (sarcasm)
When I write `ignore` I mean that the bot shouldn't close it

**Comment by robwalkerco** [comment]
This has been resolved in version 3.0.0
